Intersect.Client/Localization/Strings.cs

```diff
@@ -76,6 +76,62 @@ public static string FormatQuantityAbbreviated(long value)
             }
         }
 
+        public static string FormatTimeAbbreviated(long milliseconds)
+        {
+            if (milliseconds == 0)
+            {
+                return string.Empty;
+            }
+
+            double timeValue;
+            string postfix;
+
+            switch (milliseconds)
+            {
+                // seconds
+                case <= 60000:
+                    timeValue = milliseconds / 1000.0;
+                    postfix = Time.Second;
+                    break;
+
+                // minutes
+                case <= 600000:
+                    timeValue = milliseconds / 60000.0;
+                    postfix = Time.Minute;
+                    break;
+
+                // hours
+                case <= 36000000:
+                    timeValue = milliseconds / 3600000.0;
+                    postfix = Time.Hour;
+                    break;
+
+                // days
+                case <= 864000000:
+                    timeValue = milliseconds / 86400000.0;
+                    postfix = Time.Day;
+                    break;
+
+                // weeks
+                case <= 6048000000:
+                    timeValue = milliseconds / 604800000.0;
+                    postfix = Time.Week;
+                    break;
+
+                default:
+                    return "OOB";
+            }
+
+            if (timeValue >= 10)
+            {
+                timeValue = Math.Floor(timeValue);
+                return timeValue + postfix;
+            }
+
+            timeValue = Math.Floor(timeValue * 10) / 10.0;
+            return timeValue.ToString("F1") + postfix;
```

Comment:
Replace with
```csharp
var formattedTimeSpan = timeInUnit.ToString($"{numericalFormat}'{suffix}");
return formattedTimeSpan;
```

Reply:
All `FormatWithSuffix()` invocations can pass `"0."` as the format string to round to the nearest whole number